infos.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="de">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Unsere liebsten Mitbewohner auf 4 Pfoten">
+  <title>Die Hauskatze</title>
+  <link rel="stylesheet" href="C:\Users\codersbay\Desktop\Webaufgaben\web-aufgabe-02-kapitel-4-5-BeWie24\stil.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="infos.html" class="aktuell">weitere Informationen</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+    <a><h2>Die Sinnesorgane</h2></a>

```

Comment:
a tag in den h2 tag
inline elemente kommen immer in ein block element (machen wir noch)
a ist ein inline element
h2 ist ein blockelement = nimmt die gesamte breite des bildschirms ein